src/main/java/com/amazonaws/services/dynamodbv2/AmazonDynamoDBLockClient.java

```diff
@@ -558,6 +565,19 @@ public LockItem acquireLock(final AcquireLockOptions options) throws LockNotGran
         }
     }
 
+    /**
+     * Returns true if the client currently owns the lock with {@param key} and {@param sortKey}. It returns false otherwise.
+     *
+     * @param key     The partition key representing the lock.
+     * @param sortKey The sort key if present.
+     * @return true if the client owns the lock. It returns false otherwise.
+     */
+    public boolean hasLock(final String key, final Optional<String> sortKey) {
+        Objects.requireNonNull(sortKey, "Sort Key must not be null (can be Optional.empty())");
+        final LockItem localLock = this.locks.get(key + sortKey.orElse(""));
```

Comment:
`hasLock()` is essentially a simplified version of `[getLock()](https://github.com/awslabs/amazon-dynamodb-lock-client/blob/master/src/main/java/com/amazonaws/services/dynamodbv2/AmazonDynamoDBLockClient.java#L929)`  where, instead of returning an expired/unexpired lock it returns a boolean. 

Since getLock() does not check the owner I assume it is fine no to do so.